tasks/Task.hpp

```diff
@@ -75,6 +75,11 @@ namespace usbl_evologics {
          */
         virtual void clearTransmissionBuffer(void);
 
+        /** Drop burst data and terminate the acoustic connection
+         *
+         */
+        virtual void clearRawDataBuffer(void);
```

Comment:
Why is this virtual?

Reply:
It's the implementation of the operation defined in the .orogen file. It automatically generated a virtual method in the BaseTask, that is overridden in Task.

Reply:
Yes, of course... Looks ok to me.

Reply:
:+1: